src/IO/AssimpLoader/AssimpCameraDataLoader.cpp

```diff
@@ -68,7 +68,7 @@ void AssimpCameraDataLoader::loadCameraData( const aiScene* scene,
     Core::Matrix4 rootMatrix;
     rootMatrix = Core::Matrix4::Identity();
     /// \todo tmp test    Core::Matrix4 frame  = loadCameraFrame( scene, rootMatrix, data );
```

Comment:
This todo should be removed. It was for testing purpose only and is not really informative about what to do (nor why).